source/settings/plugin/fts-plugin.rst

```diff
@@ -54,7 +54,7 @@ Example Setting:
 
 To exclude a mailbox from automatic indexing, it can be listed in this setting.
 To exclude additional mailboxes, add sequential numbers to the end of the plugin name.
-It's possible to use either mailbox names or refer to them using special-use flags (e.g. \Trash)
+It's possible to use either mailbox names or refer to them using special-use flags (e.g. \\Trash)
```

Comment:
why not use ``\Trash`` ?

Reply:
I was wondering whether to change it to that or not. Okay, changed. Also did it for \\Recent.